code/game/objects/items/wayfinding.dm

```diff
@@ -155,57 +157,48 @@
 		user.put_in_hands(P)
 		P.owner = user.real_name
 
-/obj/machinery/pinpointer_dispenser/attackby(obj/item/I, mob/user, params)
+/obj/machinery/pinpointer_dispenser/attackby(obj/item/I, mob/living/user, params)
 	if(machine_stat & (BROKEN|NOPOWER))
 		return ..()
 
 	if(istype(I, /obj/item/pinpointer/wayfinding))
 		var/obj/item/pinpointer/wayfinding/WP = I
 
-		to_chat(user, "<span class='notice'>You put \the [WP] in the return slot.</span>")
-
-		var/refundiscredits = FALSE
 		var/itsmypinpointer = TRUE
-
-		//Will they meet the conditions to get a credit reward for recycling?
 		if(WP.owner != user.real_name)
 			itsmypinpointer = FALSE
 
-			if(synth_acc.has_money(refund_amt) && !WP.roundstart) //can it afford to refund and is the pinpointer not from the quirk
-				refundiscredits = TRUE
-				qdel(WP)
-				synth_acc._adjust_money(-refund_amt)
-				var/obj/item/holochip/holochip = new (loc)
-				holochip.credits = refund_amt
-				holochip.name = "[holochip.credits] credit holochip"
-				if(ishuman(user))
-					var/mob/living/carbon/human/customer = user
-					customer.put_in_hands(holochip)
-
-		if(!refundiscredits)
-			qdel(WP)
+		var/is_a_thing = "are [refund_amt] credit\s."
+		if(refund_amt > 0 && synth_acc.has_money(refund_amt) && !WP.roundstart)
+			synth_acc._adjust_money(-refund_amt)
```

Comment:
Don't call internal methods like this.

```suggestion
			synth_acc.adjust_money(-refund_amt)
```

Reply:
Isn't that what the proc is called?

Reply:
No.

Reply:
You seem to have a lot of assumptions driving your reaction to this PR and how you're responding to me at the moment.

That's Goof's proc he added when he made economy a couple of years ago.

Reply:
I don't know what you're talking about. `_adjust_money` is the internal proc--`adjust_money` is the one you should be calling.

![image](https://user-images.githubusercontent.com/35135081/108036100-a409c200-6fec-11eb-86ae-37e379db7b54.png)

Reply:
I was wrong but I still don't think you're coming at this from a fair-minded place. I haven't had any issues with other maintainers. So you trying to generalise it and make it only about me isn't fair.

Reply:
I have no idea what you're talking about, I didn't even know this feature existed until today, I don't have any biases towards it or you.

Reply:
Maybe I'm just reading something into it that isn't there then.